locales/locales/en.json

```diff
@@ -2390,6 +2390,9 @@
 		"message": "your votes for $1: $2",
 		"description": "As in `your votes for username: 42`"
 	},
+	"voteEnhancementsEstimated": {
+		"message": "±$1 (estimated from fuzzed score and upvote percentage)"
```

Comment:
This could imply greater precision than there actually is, should I change it to `± >1234 ...` or similar?